build.sbt

```diff
@@ -1,16 +1,25 @@
 import bintray.Keys.{repository, bintrayOrganization, bintray}
-
-scalaJSSettings
-
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
```

Comment:
Speaking of imports, these are not necessary: we're in a `.sbt`, where they are imported automatically.